astropy/io/ascii/tests/test_write.py

```diff
@@ -339,7 +340,7 @@
          ),
 ]
 
-test_def_masked_fill_value = [
+test_defs_masked_fill_value = [
```

Comment:
Why this change? It sees to me that "test_def" is jsut as good as "test_defs" here.